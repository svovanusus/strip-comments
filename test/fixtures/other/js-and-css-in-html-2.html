<style>
  /* This is style block */
  body {
    background: #fafafa;
    /*padding: 5px;*/
    margin: 8px;
  }
</style>

<style type="text/css" scoped>
  /* This is second style block */
  .class1 {
    background: red;
  }
</style>

<div>
  <p>Some content</p>
</div>

<script type="text/javascript" async>
  var x = 5; //this is init value for x
  /*
   * This is block
   * comment
   */

  for (var i = 0; i < x; i++) {
    console.log(i); // We should write iterator value here...
  }

  [].map(y => Number(y)).forEach(item => {
    console.log('this is item' + item + '. Url = https://google.com');
  });
</script>
