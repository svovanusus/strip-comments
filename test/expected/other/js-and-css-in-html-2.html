<style>
    body {
    background: #fafafa;
        margin: 8px;
  }
</style>

<style type="text/css" scoped>
    .class1 {
    background: red;
  }
</style>

<div>
  <p>Some content</p>
</div>

<script type="text/javascript" async>
  var x = 5; 
  
  for (var i = 0; i < x; i++) {
    console.log(i); 
  }

  [].map(y => Number(y)).forEach(item => {
    console.log('this is item' + item + '. Url = https://google.com');
  });
</script>
